src/aks-preview/HISTORY.md

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.5.53
+++++++
+* Add preview support for agentpools provisioned on Azure Dedicated Host Group. Requires registering the feature flag "Microsoft.ContainerService/DedicatedHostGroupPreview" for setting agentpool property "host-group-id".
```

Comment:
I guess it would be better to explicitly point out that `az aks create` and `az aks nodepool add` commands support this new option `--host-group-id`?